### PR TITLE
Fix dragon sorcerer bloodlines and dragon disciple interaction

### PR DESCRIPTION
**Pull request by Minniehajj**
This fixes an issue we learned with Draconic Bloodlines.
Before this change, if you were a draconic or wyrmblessed sorcerer, your dragon choices did not properly carry over in other places that mattered, such as the Dragon Disciple Dedication

### CODE REVIEW

**Comment by TikaelSol** [comment]
I put the title in imperative case. PR titles should finish the sentence "If accepted this PR will ____" so that the title can be used directly in the changelog

File: packs/classfeatures/bloodline-draconic.json [hunk 1]
```diff
@@ -336,4 +341,4 @@
         }
     },
     "type": "feat"
-}
+}
```
Comment (TikaelSol) [inline packs/classfeatures/bloodline-draconic.json:344]:
```suggestion
}

```

File: packs/classfeatures/bloodline-wyrmblessed.json [hunk 2]
```diff
@@ -336,4 +341,4 @@
         }
     },
     "type": "feat"
-}
+}
```
Comment (TikaelSol) [inline packs/classfeatures/bloodline-wyrmblessed.json:344]:
```suggestion
}

```

**Comment by Minniehajj** [comment]
@TikaelSol awesome, thank you! Sorry, used to semantic commit messages and PRs!

File: packs/classfeatures/bloodline-wyrmblessed.json (hunk 2 above)
Comment (Minniehajj) [reply packs/classfeatures/bloodline-wyrmblessed.json]: did I mess up a prettier config somewhere?

**Pull request merged**